policy/manager.go

```diff
@@ -48,16 +58,23 @@ func NewManager(log hclog.Logger, ps map[SourceName]Source, pm *manager.PluginMa
 
 	return &Manager{
 		log:             log.ResetNamed("policy_manager"),
-		policySource:    ps,
-		pluginManager:   pm,
-		handlers:        make(map[PolicyID]*Handler),
-		keep:            make(map[PolicyID]bool),
+		policySources:   ps,
+		targetGetter:    pm,
+		hanldersLock:    sync.RWMutex{},
```

Comment:
It doesn't matter, but you don't actually need to initialize this; the default value for a mutex is initialized.